### DISCUSSION TITLE
Linked-in Profile Link

### DISCUSSION THREAD

**Discussion by NihadZeidan** [dailydotdev/daily#1191] [Feature Requests]
I think it will beneficial to see dedicated area to add **Linked-in** link inside your profile, as it considered professional platform for any employee/company, and it will make it helpful for us to connect with each other.

![image](https://github.com/dailydotdev/daily/assets/77915170/9d44a945-bec8-40a2-80fa-8a3ef7b8ff86)

**Comment by idoshamun** [comment]
Hi @NihadZeidan,

Thanks for the suggestion, we'll consider it 🙏
